airflow/docker.go

```diff
@@ -200,18 +200,6 @@ func DockerComposeInit(airflowHome, envFile, dockerfile, imageName string) (*Doc
 //
 //nolint:gocognit
 func (d *DockerCompose) Start(imageName, settingsFile, composeFile, buildSecretString string, noCache, noBrowser bool, waitTime time.Duration, envConns map[string]astrocore.EnvironmentObjectConnection) error {
-	// check if docker is up for macOS
-	containerRuntime, err := GetContainerRuntimeBinary()
-	if err != nil {
-		return err
-	}
-	if runtime.GOOS == "darwin" && containerRuntime == dockerCmd {
-		err := startDocker()
-		if err != nil {
-			return err
-		}
-	}
-
```

Comment:
This has been moved to a cobra `PersistentPreRunE` function named `ConfiguredContainerRuntime` and a `PreRunE` function named `EnsureRuntime`. These will be rolled out more broadly across the `astro dev` commands in the next PR.